### DISCUSSION TITLE
在WPF中的Xaml为什么老是说找不到我的这个类

### DISCUSSION THREAD

**Discussion by yqyq666** [BYJRK/DotNet-Discussions#373] [C# 技术交流]
![image](https://github.com/user-attachments/assets/a4129c97-6427-41e1-b4f9-a8a80cf00900)
![image](https://github.com/user-attachments/assets/793a83d7-77c9-4f28-9fbb-ee5bdc54a957)

**Comment by CodingOctocat** [comment]
加上 Type 看看。
`d:DataContext="{d:DesignInstance Type=vm:ViewModel, IsDesignTimeCreatable=True}"`
